tests/components/rest/test_sensor.py

```diff
@@ -677,3 +679,50 @@ def test_update_request_exception(self, mock_req):
         """Test update when a request exception occurs."""
         self.rest.update()
         assert self.rest.data is None
+
+
+async def test_reload(hass):
+    """Verify we can reload reset sensors."""
+
+    with requests_mock.Mocker() as mock_req:
```

Comment:
The requests mocker is available as a pytest fixture:
https://github.com/home-assistant/core/blob/eaac00acfcaa1dac1286eb184416a6bed17a3238/tests/conftest.py#L158-L162

Reply:
Adjusted here https://github.com/home-assistant/core/pull/39282